package.json

```diff
@@ -3,10 +3,21 @@
   "version": "1.1.0",
   "description": "Turbocharged JSON.parse for type stable JSON data",
   "main": "index.js",
+  "scripts": {
+    "unit": "tap test.js",
```

Comment:
jest maybe?

Reply:
Again here. I don't force anyone to adopt `tap`.

Reply:
Yes that's okay. I'm suggesting a more robust/popular tool here and above, but of course it's up to the maintainer to decide.